[2.0.2] en: add two keepalive config in worker config file (#580)
pingcap/docs-dm: en/dm-worker-configuration-file.md

advertise-addr = "127.0.0.1:8262"
join = "http://127.0.0.1:8261,http://127.0.0.1:8361,http://127.0.0.1:8461"

keepalive-ttl = 60

> relay-keepalive-ttl = 1800 # New in v2.0.2.

Reviewer: ```suggestion
relay-keepalive-ttl = 1800 # New in DM v2.0.2.
```

| `worker-addr` | Specifies the address of DM-worker which provides services. You can omit the IP address and specify the port number only, such as ":8262". |
| `advertise-addr` | Specifies the address that DM-worker advertises to the outside world. |
| `join` | Corresponds to one or more [`master-addr`s](dm-master-configuration-file.md#global-configuration) in the DM-master configuration file. |

> | `keepalive-ttl` | The keepalive time (in seconds) of a DM-worker node to the DM-master node when the upstream data source it binds does not enable the relay log. The default value is 60s.|

Reviewer: 1. In the sentence, what does "it" represents, the DM-worker node, or the DM-master node? @lichunzhu , please help confirm. 
2. Suggest updating "when" to "if". 

```suggestion
| `keepalive-ttl` | The keepalive time (in seconds) of a DM-worker node to the DM-master node if the upstream data source it binds does not enable the relay log. The default value is 60s.|
```

Author: "it" represents the DM-worker node. Updated in https://github.com/pingcap/docs-dm/pull/580/commits/21b594be2bf955f780b9eecb6176d5e7da5e296e.

> | `relay-keepalive-ttl` | The keepalive time (in seconds) of a DM-worker node to the DM-master node when the upstream data source it binds enables the relay log. The default value is 1800s. New in v2.0.2.|

Reviewer: 1. In the sentence, what does "it" represents, the DM-worker node, or the DM-master node? @lichunzhu , PTAL. Thanks.
2. Suggest updating "when" to "if".
```suggestion
| `relay-keepalive-ttl` | The keepalive time (in seconds) of a DM-worker node to the DM-master node when the upstream data source it binds enables the relay log. The default value is 1800s. New in v2.0.2.|
```

> | `relay-keepalive-ttl` | The keepalive time (in seconds) of a DM-worker node to the DM-master node if the upstream data source of the DM-worker node enables the relay log. The default value is 1800s. New in DM v2.0.2.|

Reviewer 2: ```suggestion
| `relay-keepalive-ttl` | The keepalive time (in seconds) of a DM-worker node to the DM-master node if the upstream data source of the DM-worker node enables the relay log. The default value is 1800s. This parameter is added from DM v2.0.2.|
```
